server/selection_test.go

```diff
@@ -496,7 +497,13 @@ func TestMinLSSelector_SelectUnknownSession_RandFreq(t *testing.T) {
 	sel.Add(sessions)
 
 	// When randFreq = 1.0 we should select randomly instead of selecting the session with the most stake
-	sess := sel.selectUnknownSession(context.TODO())
+	var sess *BroadcastSession
+	for i:=0; i<10; i++ {
+		sess = sel.selectUnknownSession(context.TODO())
+		if !cmp.Equal(sess.OrchestratorInfo.TicketParams.Recipient, addr.Bytes()) {
```

Comment:
We should be able to use [bytes.Equal](https://pkg.go.dev/bytes#Equal) from the Go stdlib instead of using another package here (note: if we used `github.com/google/go-cmp/cmp` then we would also need to update the `go.mod` and `go.sum` files to track the package version used).

Reply:
f16de03ba49817a7f49b9313a7f52ef8bb1e0a4c swaps `bytes.Equal` in for the `cmp.Equal` call using `go-cmp`.